doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1360,17 +1360,17 @@ wrong loadpath afterwards.
 Some folders have a special status and are automatically put in the
 path. |Coq| commands associate automatically a logical path to files in
 the repository trees rooted at the directory from where the command is
-launched, coqlib/user-contrib/, the directories listed in the
-`$COQPATH`, `${XDG_DATA_HOME}/coq/` and `${XDG_DATA_DIRS}/coq/`
-environment variables (see`http://standards.freedesktop.org/basedir-
-spec/basedir-spec-latest.html`_) with the same physical-to-logical
-translation and with an empty logical prefix.
+launched, ``coqlib/user-contrib/``, the directories listed in the
+``$COQPATH``, ``${XDG_DATA_HOME}/coq/`` and ``${XDG_DATA_DIRS}/coq/``
+environment variables (see `XDG base directory specification
+<http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html>`_)
+with the same physical-to-logical translation and with an empty logical prefix.
 
 The command line option ``-R`` is a variant of ``-Q`` which has the strictly
 same behavior regarding loadpaths, but which also makes the
 corresponding ``.vo`` files available through their short names in a way
 not unlike the ``Import`` command (see :ref:`here <import_qualid>`). For instance, ``-R`` `path` ``Lib``
```

Comment:
You could also improve this line, by using ```:cmd:`Import` ``` and removing the parenthesis.
That being said, I don't personally understand this piece of sentence at all and wonder whether it wouldn't be better to just remove "in a way not unlike the Import command".